dali/pipeline/pipeline.cc

```diff
@@ -182,16 +182,16 @@ void Pipeline::AddOperator(OpSpec spec, const std::string& inst_name) {
   this->op_specs_for_serialization_.push_back(make_pair(inst_name, spec));
 
   // If necessary, split nvJPEGDecoder operator in two separated stages (CPU and Mixed-GPU)
-#ifdef NVJPEG_DECOUPLED_API
```

Comment:
This change allows explicitly disabling `NVJPEG_DECOUPLED_API` in `dali/pipeline/operators/decoder/CMakeLists.txt`, without affecting the whole project - faster build when switching between APIs.